src/SFA.DAS.ApprenticeFeedback.Api/AppStart/NServiceBusRegistration.cs

```diff
@@ -67,7 +68,8 @@ private static DbConnection CreateSqlConnection(IConfiguration configuration, Ap
                 return new SqlConnection
                 {
                     ConnectionString = appSettings.DbConnectionString,
-                    AccessToken = new AzureServiceTokenProvider().GetAccessTokenAsync("https://database.windows.net/").Result
+                    AccessToken = new ChainedTokenCredential().GetTokenAsync(
+                        new TokenRequestContext(new string[] { "https://database.windows.net/" })).Result.Token
```

Comment:
I'm not sure about this one. Here I've used [this PR](https://github.com/SkillsFundingAgency/das-apim-developer-api/pull/3/files#diff-8a6c98ca86a8d24b1ed7d3f79b38737bbdfacbb14091d4dbfbb47244fd76e5e9) as a guide and used `ChainedTokenCredential`, but the [official migration guide](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/app-auth-migration?view=azure-dotnet) suggests using `DefaultAzureCredential`, and makes no mention of the `ChainedTokenCredential` used in the other PR.

Which is the correct way?

Reply:
What you've done will work - however the _correct_ way would be to just use the DefaultAzureCredential and allowing the details of the connection string to determine how it obtains the access token.

Reply:
I've changed it to use DefaultAzureCredential, does it look alright?